staging/src/k8s.io/kubectl/pkg/cmd/logs/logs.go

```diff
@@ -319,6 +319,10 @@ func (o LogsOptions) RunLogs() error {
 		return err
 	}
 
+	if len(requests) == 0 {
+		return fmt.Errorf("no resources found in %s namespace", o.Namespace)
```

Comment:
Question because I’m not super familiar with this part, but is this the correct message?

For example, is it possible for the resource to exist but have no logs available, and if so, wouldn’t the message be more accurate to say something like “no logs found for resources in %s namespace”

I’m ok with the message staying this way if it is accurate.  I just wanted to ask the question...

Reply:
@brianpursley 
This PR only consider about no resource to exist like get or describe command.

In this case(resource to exist but have no logs available), it doesn't print error message in this PR. 
I've been thinking about resource to exist but have no logs available but I don't know how to deal with follow option. So I don't care about it.